tests/optional_kernel_features/kernel_features_common.h

```diff
@@ -494,17 +905,23 @@ class kernel_submission_call;
                                                                             \
   {                                                                         \
     auto single_task_action = [&QUEUE] {                                    \
+      bool value = false;                                                   \
```

Comment:
Some day we will have to remove all these macros... :-(

Reply:
I hope.